### PR TITLE
Update icon colors for wcag 2.1

### PR DESCRIPTION
**Pull request by dbatiste**
Updates icons with new HEX values for:
* ferrite
* mica
* sylvite

Note for follow-up: the new [icon.js](https://github.com/BrightspaceUI/core/blob/master/components/icons/icon.js#L143) logic looks for `path[fill]` and removes those attributes.  In effect it will likely result in ferrite (inherited from ancestor) instead of the previous mica or sylvite colors.  We can potentially update to a more specific check, but the multi-color icon makes it difficult to override the color if desired.  The icon should be one color to properly enable that.

### CODE REVIEW

File: components/icons/images/emoji/angry.svg [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
-    <path fill="#565a5c" d="M9 0a9 9 0 1 0 9 9 9 9 0 0 0-9-9zm1.29 5.29l2-2a1 1 0 0 1 1.6 1.16l-.68 1.34.51.51a1 1 0 0 1 .09 1.29A1.028 1.028 0 0 1 13 8a.934.934 0 0 1-.45-.11l-2-1a1.009 1.009 0 0 1-.55-.73 1.038 1.038 0 0 1 .29-.87zm-6.18-.84a1 1 0 0 1 1.6-1.16l2 2a1.032 1.032 0 0 1 .28.87 1 1 0 0 1-.54.73l-2 1A.934.934 0 0 1 5 8a1.028 1.028 0 0 1-.81-.41 1 1 0 0 1 .09-1.29l.51-.51zM14 12H4a1 1 0 0 1 0-2h10a1 1 0 0 1 0 2z"/>
+    <path fill="#494C4E" d="M9 0a9 9 0 1 0 9 9 9 9 0 0 0-9-9zm1.29 5.29l2-2a1 1 0 0 1 1.6 1.16l-.68 1.34.51.51a1 1 0 0 1 .09 1.29A1.028 1.028 0 0 1 13 8a.934.934 0 0 1-.45-.11l-2-1a1.009 1.009 0 0 1-.55-.73 1.038 1.038 0 0 1 .29-.87zm-6.18-.84a1 1 0 0 1 1.6-1.16l2 2a1.032 1.032 0 0 1 .28.87 1 1 0 0 1-.54.73l-2 1A.934.934 0 0 1 5 8a1.028 1.028 0 0 1-.81-.41 1 1 0 0 1 .09-1.29l.51-.51zM14 12H4a1 1 0 0 1 0-2h10a1 1 0 0 1 0 2z"/>
```
Comment (dlockhart) [inline components/icons/images/emoji/angry.svg]: Nit: we've been using all lowercase for the HEX colours everywhere I think.

**Comment by dbatiste** [comment]
Will fix the readme.

Ah, so for the fill, the following checkbox icons (which will likely need to be updated per Sarah's checkbox updates) use the Mica and Sylvite colors.  The code will remove that fill, forcing to Ferrite as you mentioned, but it looks like that's not what we want it to do.  She currently has the revised border color as Galena (from Mica), and background color Regolith.

https://github.com/BrightspaceUI/core/blob/master/components/icons/images/tier2/check-box-unchecked.svg
https://github.com/BrightspaceUI/core/blob/master/components/icons/images/tier2/check-box.svg

**Comment by dlockhart** [comment]
Ah interesting, I'm with you now. So I guess we should update icons to only strip fills that are ferrite? I can make that change separately if that's what makes sense.

**Comment by dbatiste** [comment]
The more I look into this, the more I think these two icons should just be ferrite.  Check-box menu-items use the tier-1 check, which is just ferrite without a box.  The tier-3 check inside a circle is just ferrite.  Plus, these two icons aren't even used.

**Comment by dbatiste** [comment]
These icons appear to be used in Content when manual completion tracking is enabled.  Going to merge this and we can follow-up to figure what we really want for these icons.

**Pull request merged**